src/access_nri_intake/source/builders.py

```diff
@@ -545,6 +544,70 @@ def parser(cls, file):
         except Exception:
             return {INVALID_ASSET: file, TRACEBACK: traceback.format_exc()}
 
+    @classmethod
+    def parse_ncfile(cls, file: str, time_dim: str = "time") -> _NCFileInfo:
```

Comment:
This is still a line-for-line copy of `BaseBuilder.parse_ncfile`, just with the time parser object changed. Is it workable to add a `time_parser` kwarg to `BaseBuilder.parse_ncfile`, and just inherit from there?

If you want clearer separation, your other option might be to come up with a Mixin solution that defines `parse_ncfile` separately (which is workable, given it's a classmethod.)

Reply:
Ahh gotcha - yeah, the first solution makes a lot more sense than what I've got here.

Reply:
Turned out to be easier from a Liskov perspective to just add the time parser as a class attribute - but I think the changes should address your points.

If you're happy, I think we should be ready to merge this in.